torchrl/trainers/loggers.py

```diff
@@ -4,9 +4,21 @@
 # LICENSE file in the root directory of this source tree.
 
 import abc
+import os
 
+from omegaconf import OmegaConf
 from torch import Tensor
 
+try:
+    import wandb
```

Comment:
To import tensorboard, we used to do it only in 
```python
def main()
    import tensorboard
```
The reason being that when we launch our parallel / distributed envs we want to have as few imports as possible (and `main` is called only in the master node/process).

Can we find a way of doing this here too, ie avoiding that each and every workers loads wandb / tensorboard when it is not needed?